include/nclistmap.h

```diff
@@ -7,13 +7,26 @@ See LICENSE.txt for license information.
 #define NCLISTMAP_H
 
 #include "nclist.h"
+<<<<<<< HEAD
```

Comment:
It appears that a merge/diff was accepted without being resolved.  @DennisHeimbigner can you providie guidance as to how we should resolve this?

Reply:
I think when I was resolving conflicts, I forget to remove that <<<<HEAD line. It should just be removed.